<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MiniPauzer</class>
 <widget class="QWidget" name="MiniPauzer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MiniPauzer</string>
  </property>
  <widget class="QPushButton" name="btn_Play">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>150</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </widget>
  <widget class="SliderBar" name="sliderBar">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>391</width>
     <height>18</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_CurTime">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_songLen">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>50</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0:00</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ChooseFolders">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose Music Folders</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>631</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SliderBar</class>
   <extends>QSlider</extends>
   <header>sliderbar.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
